src/js/messaging/containers/Thread.jsx

```diff
@@ -328,9 +345,17 @@ export class Thread extends React.Component {
       return <p>Sorry, this message does not exist.</p>;
     }
 
+    // check if any senderIds are valid recipients
+    const disabled = intersection(
+      recipients.map(e => e.value),
```

Comment:
Good catch. I guess it is theoretically possible. The loading now happens in the parent container `Main` so it is unlikely that somebody will be able to navigate into a message thread before the recipients have loaded, but I will add a check just in case.